src/parsers/dr_ste_v1_tag_parser.py

```diff
@@ -0,0 +1,424 @@
+from src.dr_utilities import dr_ste_v1_lu_type
+from src.dr_utilities import _val
+from src.parsers.dr_ste_v0_tag_parser import *
```

Comment:
list the 4 that you are using instead of "*"

Reply:
from src.parsers.dr_ste_v0_tag_parser import mlx5_ifc_ste_v0_general_purpose_bits_tag_parser,  mlx5_ifc_ste_v0_eth_l3_ipv6_dst_bits_tag_parser_p, mlx5_ifc_ste_v0_mpls_bits_tag_parser, mlx5_ifc_ste_v0_register_0_bits_tag_parser, mlx5_ifc_ste_v0_register_1_bits_tag_parser, mlx5_ifc_ste_v0_eth_l3_ipv6_src_bits_tag_parser_p